Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PI.mo

```diff
@@ -0,0 +1,70 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.Amigo;
+block PI "Identifies the parameters of a PI controller"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput kp(min=1E-6)
+    "Connector for the signal of the gain of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput T(min=0)
+    "Connector for the signal of the time constant of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput L(min=1E-6)
+    "Connector for the signal of the time delay of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+        iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput k
+    "Connector for control gain signal"
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput Ti
+    "Connector for time constant signal for the integral term"
+    annotation (Placement(transformation(extent={{100,-50},{120,-30}})));
```

Comment:
For some reason, your output connectors are smaller than the input connectors. For example in this instance, it is `extent={{100,-50},{120,-30}}`, which means the dimension is 20 by 20. It should be 40 by 40. 
I see in other classes, the output connector are smaller. Please resize them.

Reply:
Modified the size of the output connectors